README.md

```diff
@@ -1,89 +1,74 @@
+  * [API Tutorials](https://www.deepdetect.com/server/docs/server_docs/): training from text, data and images, setup of prediction services, and export to external software (e.g. ElasticSearch)
+  * [API Reference](http://www.deepdetect.com/api/)
+  * [Examples](http://www.deepdetect.com/overview/examples/): MLP for data, text, multi-target regression to CNN and GoogleNet, finetuning, etc...)
+  * [FAQ](http://www.deepdetect.com/overview/faq/)
+* Demos:
+  * [Image classification Web application](https://github.com/jolibrain/deepdetect/tree/master/demo/imgdetect) using HTML and javascript
+  * [Image similarity search](https://github.com/jolibrain/deepdetect/tree/master/demo/imgsearch) using python client
+  * [Image object detection](https://github.com/jolibrain/deepdetect/tree/master/demo/objdetect) using python client
+  * [Image segmentation](https://github.com/jolibrain/deepdetect/tree/master/demo/segmentation) using python client
+* [Performance tools and report](https://github.com/jolibrain/dd_performances) done on NVidia Desktop and embedded GPUs, along with Raspberry Pi 3.
+* [References](#references)
+* [Authors](#authors)
+
+## Machine Learning functionalities per library
+
+|                   | Caffe | Caffe2 | XGBoost | TensorRT | NCNN | Libtorch | Tensorflow | T\-SNE | Dlib |
+|------------------:|:-----:|:------:|:-------:|:--------:|:----:|:--------:|:----------:|:------:|:----:|
+| **Stage**         |       |        |         |          |      |          |            |        |      |
+| Training \(CPU\)  | Y     | Y      | Y       | N/A      | N/A  | Y        | N          | Y      | N    |
+| Training \(GPU\)  | Y     | Y      | Y       | N/A      | N/A  | Y        | N          | Y      | N    |
+| Inference \(CPU\) | Y     | Y      | Y       | Y        | Y    | Y        | Y          | N/A    | Y    |
+| Inference \(GPU\) | Y     | Y      | Y       | Y        | Y    | Y        | Y          | N/A    | Y    |
+| **Task**          |       |        |         |          |      |          |            |        |      |
+| Classification    | Y     | Y      | Y       | Y        | Y    | Y        | Y          | N/A    | Y    |
+| Object Detection  | Y     | Y      | N       | Y        | Y    | N        | N          | N/A    | Y    |
+| Segmentation      | Y     | N      | N       | N        | N    | N        | N          | N/A    | N    |
+| Regression        | Y     | N      | Y       | N        | N    | N        | N          | N/A    | N    |
+| Autoencoder       | Y     | N      | N/A     | N        | N    | N        | N          | N/A    | N    |
+| OCR / Seq2Seq     | Y     | N      | N       | N        | Y    | N        | N          | N      | N    |
+| Time\-Series      | Y     | N      | N       | N        | Y    | N        | N          | N      | N    |
+| **Input Data**    |       |        |         |          |      |          |            |        |      |
```

Comment:
Small detail: I'd add the header again in between sections, to ease the reading of the table, or break the table down into three tables.